# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/gc/GCEnum.h
# Commit: b622095c76ab
# Full Hash: b622095c76ab52fdd32c1fa41a1541f3c8b49d01
# Author: caroline <cullen.caroline@gmail.com>
# Date: 2019-12-10 21:29:05
# Regressor Bug: 1588861
# File Overlap Count: 1
# Description:
#   Bug 1588861 - Part 4: Add test harness for xdr modules. r=iain
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D55596
# ==============================================================================

diff -r 2a0aacbd11de -r b622095c76ab js/src/gc/GCEnum.h
--- a/js/src/gc/GCEnum.h	Tue Dec 10 02:52:37 2019 +0000
+++ b/js/src/gc/GCEnum.h	Tue Dec 10 01:46:29 2019 +0000
@@ -142,7 +142,8 @@
   _(FinalizationGroupRegistrations)        \
   _(FinalizationRecordVector)              \
   _(ZoneAllocPolicy)                       \
-  _(SharedArrayRawBuffer)
+  _(SharedArrayRawBuffer)                  \
+  _(XDRBufferElements)
 
 #define JS_FOR_EACH_MEMORY_USE(_)  \
   JS_FOR_EACH_PUBLIC_MEMORY_USE(_) \